src/azul/downloader.py

```diff
@@ -10,139 +10,40 @@
 that can be used by the Indexer subclasses.
 """
 
-from concurrent.futures import ThreadPoolExecutor
 import logging
-import os
 
-from hca.dss import SwaggerAPIException
+from humancellatlas.data.metadata.helpers.dss import download_bundle_metadata
 
 from azul import config
 
 log = logging.getLogger(__name__)
 
 
 class MetadataDownloader(object):
```

Comment:
I think this was sort of pointless even before this PR because it was instantiated and a single method was called on it before the instance went out of scope. That's smells of an anti-pattern.

Anyways, now it really is anemic and I think it should go away.